model/riscv_sys_regs.sail

```diff
@@ -161,25 +167,20 @@ function have_privLevel(priv : priv_level) -> bool =
     0b11 => true,
   }
 
-bitfield Mstatush : bits(32) = {
-  MBE  : 5,
-  SBE  : 4
-}
-register mstatush : Mstatush
-
-bitfield Mstatus : xlenbits = {
+bitfield Mstatus : bits(64) = {
   SD   : xlen - 1,
```

Comment:
This isn't where SD is on RV32 annoyingly. See #639 

We should make a separate MR to fix just #639 ideally.